src/appleseed/renderer/modeling/aov/cryptomatteaov.cpp

```diff
@@ -510,11 +510,17 @@ namespace
                         pixel_values.push_back(b);
 
                         // Remove background contribution.
+                        size_t ranked_vector_start = 0;
+                        size_t ranked_vector_size = ranked_vector.size();
+
                         if (ranked_vector.size() > 1 && m3hash_preview == 0)
```

Comment:
You could use `ranked_vector_size` here.

Reply:
Good point